### ISSUE TITLE
[Accessibility Audit] Batch Changes: (Admin) Add a global access token to use with batch changes

### ISSUE THREAD

**Issue by umpox** [sourcegraph/sourcegraph#34093]
### Steps to replicate journey

Taken from the [user journey list document](https://docs.google.com/document/d/1XgG0udYL37G3RxuT5shizulpHgb6xVtY5L9kBeSkXYk/edit?usp=sharing). Use this for further context.

- Click your user profile avatar in the top right corner
- Click “Site admin” from the dropdown menu
- Click “Settings” from the sidebar, within the “Batch Changes” section
- Click “Add credentials” for a code host without global credentials configured (has a red empty circle next to the code host name)
- Follow the steps to create a new personal access token on the code host
  - Note: We do not need to audit these steps as we cannot ensure the WCAG status of external sites.
- Paste the personal access token into the input field
- Click “Add credential”
- Wait for the modal to close
- A green checkbox should appear next to the code host name

### How to audit

Follow the instructions here: [Auditing a user journey](https://docs.sourcegraph.com/dev/background-information/web/accessibility/how-to-audit#auditing-a-user-journey)

**Note:** We are planning on arranging thorough third-party audit in the future, so our focus here is just to find the _majority_ of accessibility issues. Don't worry if you feel you haven't found 100% of potential issues, it is better to focus on the core essentials to complete the journey rather than spending lots of time going through every possible problem.

### How to raise discovered problems

Follow the instructions here: [Raising an accessibility bug](https://docs.sourcegraph.com/dev/background-information/web/accessibility/how-to-audit#raising-a-bug)

### Any other questions?

See the [audit FAQ](https://docs.google.com/document/d/1nCi7TvYRpj9eG0_QtgqK1c4L28EiJKVuxyuvg1lAGgo/edit#) - add your question if it isn't there already!

**Comment by courier-new** [comment]
Closing as all issues with this journey have already been identified from other journeys.
